docs/iidm/exporter/iidm.md

```diff
@@ -106,9 +120,32 @@ The export mode can be:
    - `IidmImportExportMode.ONE_SEPARATED_FILE_PER_EXTENSION_TYPE`: in case we want to export network in a file and each extension type in a separate file.
    Example: if our network `test` has two extensions `loadFoo` and `loadBar` then the network will be exported 
     in the `test.xiidm` file when `loadFoo` and `loadBar` will be exported respectively in `test-loadFoo.xiidm` and `test-loadBar.xiidm`.
-
 The default value for this parameter is `IidmImportExportMode.NO_SEPARATED_FILE_FOR_EXTENSIONS`.
 
+## iidm.export.xml.incremental-conversion
+The `iidm.export.xml.incremental-conversion` property is an optional property that defines whether the export type is 
+incremental or not.
+
+Setting that attribute to true lets us export only the elements having control, state or topology attributes.
+we export those elements to three separated files `base-TOPO.xiidm`, `base-STATE.xiidm` and `base-CONTROL.xiidm`.
+
+The default value for this parameter is `false`.
+
```

Comment:
The `iidm.export.xml.incremental-conversion` property is an optional property that defines whether the export type is incremental or not. Incremental means here that only network variables are exported. Note that this export is not a full export, contrary to the three previous modes, described below. Setting that attribute to `true` lets us export only the elements having control, state or topology attributes. These elements are exported into three separated files (by default):

   - `base-STATE.xiidm` file: contains only network elements having state attributes.

   - `base-TOPO.xiidm` file: contains only network elements having topology attributes.

   - `base-CONTROL.xiidm` file: contains only network elements having control attributes.

The default value for this parameter is `false`.

Reply:
It's easier to write the paragraph for me because I really changed the introduction, so it was to transfert your words here.